yunwoo/baekjoon/수학/2869-달팽이는-올라가고-싶다.js

```diff
@@ -0,0 +1,56 @@
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const input = [];
+
+rl.on("line", function (line) {
+  input.push(line);
+}).on("close", function () {
+  // 풀이
+  const [a, b, v] = input[0].split(" ").map(Number);
+  const day = (v - b) / (a - b);
+  const result = Number.isInteger(day) ? day : Math.ceil(day);
+
+  console.log(result);
+});
```

Comment:
```suggestion
const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

const input = [];

rl.on("line", function (line) {
  input.push(line);
}).on("close", function () {
  // 풀이
  const [a, b, v] = input[0].split(" ").map(Number);
  const result = Math.ceil((v - b) / (a - b));

  console.log(result);
});
```
- 풀이 잘봤습니다 🙇🏻‍♂️ day를 애초에 Math.ceil로 감싸주면 아래에서 삼항 연산자를 하실 필요가 없습니다~!

Reply:
- 오!! 좋은 방법 감사합니다🙇‍♂️